src/zjs_i2c.c

```diff
@@ -73,17 +73,23 @@ static jerry_value_t zjs_i2c_read_base(const jerry_value_t this,
     }
 
     if (!burst) {
-        uint8_t error_msg = zjs_i2c_handle_read((uint8_t)bus,
-                                                buf->buffer,
-                                                buf->bufsize,
-                                                (uint16_t)address);
+        int error_msg = zjs_i2c_handle_read((uint8_t)bus,
+                                            buf->buffer,
+                                            buf->bufsize,
+                                            (uint16_t)address);
+        if (error_msg != 0) {
+            ERR_PRINT("i2c_read failed with error %i\n", error_msg);
```

Comment:
I'd like to give the user the ability to recover and try again if it should i2c_read fail.  It happens (rarely) but I have seen it happen over long periods of time.  That's why I have an ERR_PRINT vs zjs_error